admin_ui_support/modules/admin_ui_support_schemata/src/Normalizer/json/SimpleConfigSchemaNormalizer.php

```diff
@@ -0,0 +1,72 @@
+<?php
+
+namespace Drupal\admin_ui_support_schemata\Normalizer\json;
+
+use Drupal\admin_ui_support_schemata\Schema\SimpleConfigSchema;
+use Drupal\admin_ui_support_schemata\SimpleConfigSchemaUrl;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\schemata_json_schema\Normalizer\json\JsonNormalizerBase;
+
+class SimpleConfigSchemaNormalizer extends JsonNormalizerBase {
+
+  /**
+   * The interface or class that this Normalizer supports.
+   *
+   * @var string
+   */
+  protected $supportedInterfaceOrClass = 'Drupal\admin_ui_support_schemata\Schema\SimpleConfigSchema';
+
+  /**
+   * {@inheritdoc}
+   */
+  public function normalize($simple_config_schema, $format = NULL, array $context = []) {
+    /* @var $simple_config_schema \Drupal\admin_ui_support_schemata\Schema\SimpleConfigSchema */
+    //@fixme
+    $generated_url = SimpleConfigSchemaUrl::fromSchema($this->format, $this->describedFormat, $simple_config_schema)
+      ->toString(TRUE);
+    // Create the array of normalized fields, starting with the URI.
+    $normalized = [
+      '$schema' => 'http://json-schema.org/draft-04/schema#',
+      'id' => $generated_url->getGeneratedUrl(),
+      'type' => 'object',
+    ];
+    $normalized = array_merge($normalized, $simple_config_schema->getMetadata());
+
+    // Stash schema request parameters.
+    $context['simple_config_name'] = $simple_config_schema->getName();
+
+    // Retrieve 'properties' and possibly 'required' nested arrays.
+    $properties = $this->normalizeProperties(
+      $this->getProperties($simple_config_schema, $format, $context),
+      $format,
+      $context
+    );
+    $normalized = NestedArray::mergeDeep($normalized, $properties);
```

Comment:
This merge actually wipes out the top level "title" element. It seems to grab "title" from some sub element but can't figure out the pattern